omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -33,6 +33,8 @@
 
 rabbitmq = OmnibusHelper.new(node).rabbitmq_configuration
 
+elastic_search_version = OmnibusHelper.new(node).elastic_search_major_version
```

Comment:
I am not sure how that happened. We even tested that change before we checked it in.